src/stan/services/arguments/singleton_argument.hpp

```diff
@@ -44,7 +44,7 @@ namespace stan {
     template<typename T>
     class singleton_argument: public valued_argument {
     public:
-      singleton_argument(): _validity("All") {
+      explicit singleton_argument(): _validity("All") {
```

Comment:
Why are you making this unary constructor explicit?  Will this still work:

```
singleton_argument a;
```

?